Makefile

```diff
@@ -1,6 +1,8 @@
 # --- Global -------------------------------------------------------------------
 O = out
 COVERAGE = 91
+SEMVER ?= v0.0.0-dev
+COMMIT_SHA ?= $(or $(shell [ -d .git ] && git rev-parse --short HEAD),0000000)
```

Comment:
I feel we can get rid of the `[ -d .git ]` stuff. We only did that for cloudbuild which 1) we're not using, and 2) they've fixed. Lets keep it simple and make this just:

    COMMIT_SHA ?= $(shell git rev-parse --short HEAD)

Reply:
actually, sorry, not an option. not working in docker (assuming we don't want to copy .git too?)

Reply:
actually, sorry, double wammy - all the docker bussiness is missing, the whole annoying arg passing... if that's what we want to do.

Reply:
fixed.